torch/_subclasses/functional_tensor.py

```diff
@@ -163,7 +163,8 @@ def __new__(cls, elem, mode):
         out.elem = elem
 
         if (
-            torch.is_inference_mode_enabled()
+            not mode.export
```

Comment:
maybe worth a quick commenting mentioning it's because of a bug, not fundamental?